mne/viz/eyetracking/heatmap.py

```diff
@@ -0,0 +1,167 @@
+# Authors: Scott Huberty <seh33@uw.edu>
+#
+# License: BSD-3-Clause
+
+import numpy as np
+from scipy.ndimage import gaussian_filter
+
+
+from ..utils import plt_show
+from ...utils import _validate_type, logger, fill_doc
+
+
+@fill_doc
+def plot_gaze(
+    epochs,
+    width,
+    height,
+    sigma=25,
+    cmap=None,
+    alpha=None,
+    vmin=None,
+    vmax=None,
+    axes=None,
+    show=True,
+):
+    """Plot a heatmap of eyetracking gaze data.
+
+    Parameters
+    ----------
+    epochs : mne.Epochs
+        The :class:`~mne.Epochs` object containing eyegaze channels.
+    width : int
+        The width dimension of the plot canvas. For example, if the eyegaze data units
+        are pixels, and the participant screen resolution was 1920x1080, then the width
+        should be 1920.
+    height : int
+        The height dimension of the plot canvas. For example, if the eyegaze data units
+        are pixels, and the participant screen resolution was 1920x1080, then the height
+        should be 1080.
+    sigma : int | float
+        The amount of smoothing applied to the heatmap data. If ``None``,
+        no smoothing is applied. Default is 1.
```

Comment:
I think this is probably fine because so far we've only been dealing with pixel data, but just raising the possibility that someone _could_ have gaze data that in visual degrees, in which case technically the `"in pixels"` part would be incorrect?